pages/Transfer.qml

```diff
@@ -195,6 +195,17 @@ Rectangle {
            ListElement { column1: qsTr("High (x166 fee)")  ; column2: "";  priority: PendingTransaction.Priority_High }
        }
 
+      // Priorites after v5
+      ListModel {
+           id: priorityModelV5
+
+           ListElement { column1: qsTr("Low (x1 fee)") ; column2: ""; priority: 1}
+           ListElement { column1: qsTr("Medium (x4 fee)") ; column2: ""; priority: 2 }
+           ListElement { column1: qsTr("High (x20 fee)") ; column2: ""; priority: 3 }
+           ListElement { column1: qsTr("Highest (x166 fee)")  ; column2: "";  priority: 4 }
```

Comment:
Seems odd to change the name-to-multiplier mapping. I'd have added a new one below low. Doesn't really matter that much though I guess.